src/sonic-yang-models/yang-models/sonic-mgmt-interface.yang

```diff
@@ -0,0 +1,72 @@
+module sonic-mgmt-interface {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-mgmt-interface";
+    prefix mgmtintf;
+
+    import sonic-mgmt-port {
+        prefix mgmtprt;
+    }
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-types {
+        prefix stypes;
+    }
+
+    description
+        "SONiC MANAGEMENT INTERFACE";
+
+    revision 2019-11-13 {
+        description "First revision";
+    }
+
+    container sonic-mgmt-interface {
+
+        container MGMT_INTERFACE {
+
+            description "MANAGEMENT INTERFACE part of config_db.json";
+
+            list MGMT_INTERFACE_LIST {
+
+                key "name";
+
+                leaf name{
+                    type leafref {
+                        path /mgmtprt:sonic-mgmt-port/mgmtprt:MGMT_PORT/mgmtprt:MGMT_PORT_LIST/mgmtprt:name;
+                    }
+                }
+            }
```

Comment:
This is for vrf association which is not applicable for mgmt intf. So maybe remove this?

Reply:
Removed.